packages/contentstack-utilities/package.json

```diff
@@ -26,26 +26,26 @@
   "author": "contentstack",
   "license": "MIT",
   "dependencies": {
-    "@contentstack/management": "^1.3.0",
+    "@contentstack/management": "^1.6.0",
     "@oclif/core": "^1.8.2",
-    "axios": "^0.27.2",
+    "axios": "1.1.3",
```

Comment:
Why are we using a specific version of axios ?

@antonyagustine

Reply:
1.2.0 breaks our utility. So we need to check for the utility breaking change before updating the current version.

CC @netrajpatel